doc/platform_api/CMIS_and_C-CMIS_support_for_ZR.md

```diff
@@ -0,0 +1,1545 @@
+  Datapath Init Pending Host Lane 8: False
+  Tuning In Progress: False
+  Wavelegnth Unlock Status : False
+  Target Output Power Out Of Range : False
+  Fine Tuning Out Of Range : False
+  Tuning Not Accepted : False
+  Invalid Channel Number : False
+  Tuning Complete : True
+  Temperature High Alarm : False
+  Temperature High Warning : False
+  Temperature Low Alarm : False
+  Temperature Low Warning : False
+  Vcc High Alarm : False
+  Vcc High Warning : False
+  Vcc Low Alarm : False
+  Vcc Low Warning : False
+  Tx Power High Alarm : False
+  Tx Power High Warning : False
+  Tx Power Low Alarm : False
+  Tx Power Low Warning : False
+  Rx Power High Alarm : False
+  Rx Power High Warning : False
+  Rx Power Low Alarm : False
+  Rx Power Low Warning : False  
+  Tx Bias High Alarm : False
+  Tx Bias High Warning : False
+  Tx Bias Low Alarm : False
+  Tx Bias Low Warning : False  
+  Laser Temperature High Alarm : False
+  Laser Temperature High Warning : False
+  Laser Temperature Low Alarm : False
+  Laser Temperature Low Warning : False
+  ```
+- Example (Display pm status of transceiver connected to Ethernet0):
+  ```
+  admin@sonic:~$ show interfaces transceiver pm Ethernet0
+  Ethernet0:
+  prefec_ber_avg : 1.00E-3
+  prefec_ber_min : 1.00E-3
+  prefec_ber_max : 1.00E-3
+  uncorr_frames_avg : 0.00E0
+  uncorr_frames_min : 0.00E0
+  uncorr_frames_max : 0.00E0
+  cd_avg : 0 ps/ns
+  cd_min : 0 ps/ns
+  cd_max : 0 ps/ns
+  dgd_avg : 1.0 ps
+  dgd_min : 1.0 ps
+  dgd_max : 1.0 ps
+  sopmd_avg : 1 ps^2
+  sopmd_min : 1 ps^2
+  sopmd_max : 1 ps^2
+  pdl_avg  : 1.0 dB
+  pdl_min  : 1.0 dB
+  pdl_max  : 1.0 dB
+  osnr_avg : 28.0 dB
+  osnr_min : 28.0 dB
+  osnr_max : 28.0 dB
+  esnr_avg : 15.0 dB
+  esnr_min : 15.0 dB
+  esnr_max : 15.0 dB
+  cfo_avg  : 500 MHz
+  cfo_min  : 500 MHz
+  cfo_max  : 500 MHz
+  soproc_avg : 1 krad/s
+  soproc_min : 1 krad/s
+  soproc_max : 1 krad/s
+  tx_power_avg : -10.00 dBm
+  tx_power_min : -10.00 dBm
+  tx_power_max : -10.00 dBm
+  rx_tot_power_avg : -8.00 dBm
+  rx_tot_power_min : -8.00 dBm
+  rx_tot_power_max : -8.00 dBm
+  rx_sig_power_avg : -8.00 dBm
+  rx_sig_power_min : -8.00 dBm
+  rx_sig_power_max : -8.00 dBm
+  ```
+
+- Example (Display loopback status of transceiver connected to Ethernet0):
+  ```
+  admin@sonic:~$ show interfaces transceiver loopback Ethernet0
+  Ethernet0:
+  Media Output Loopback : False
+  Media Input Loopback : False
+  Host Output Loopback : False
+  Host Input Loopback : False
+  ```
+
+#### 2.3 Config_DB Schema ####
+##### 2.3.1 Transceiver Config Table #####
+Stores information for physical switch ports managed by the switch chip. Ports to the CPU (ie: management port) and logical ports (loopback) are not declared in the PORT_TABLE. See INTF_TABLE.
+
+    ;Defines layer 2 ports
+    ;In SONiC, Data is loaded from configuration file by portsyncd
+    key                 = PORT_TABLE:ifname     ; ifname must be unique across PORT,INTF,VLAN,LAG TABLES
```

Comment:
in the port split mode, there could be multi interfaces sitting on top of one cable, how do you want to handle this case, especially when the configuration to the same cable is not aligned?

Reply:
I am not sure if I understand the question correctly. There will be different host/media lanes mapping to multiple interfaces with different configurations.